sequencer/src/api/fs.rs

```diff
@@ -1,17 +1,14 @@
-use super::{
-    data_source::{Provider, SequencerDataSource},
-    endpoints::TimeWindowQueryData,
-};
+use super::data_source::{Provider, SequencerDataSource};
 use crate::{persistence::fs::Options, SeqTypes};
 use async_trait::async_trait;
 use futures::StreamExt;
 use hotshot_query_service::{
-    availability::{AvailabilityDataSource, BlockId, BlockQueryData},
+    availability::{AvailabilityDataSource, BlockQueryData},
     data_source::{ExtensibleDataSource, FileSystemDataSource},
     node::NodeDataSource,
-    NotFoundSnafu, QueryError, QueryResult,
+    types::HeightIndexed,
+    QueryError,
 };
-use snafu::OptionExt;
 use std::{collections::BTreeMap, path::Path};
 
 #[derive(Clone, Debug, Default)]
```

Comment:
I think we can also remove `blocks_by_time` and switch from `ExtensibleDataSource` to plain `FileSystemDataSource`

Reply:
I also removed `refresh_indices`